src/_guides/libraries/c-interop.md

```diff
@@ -184,27 +184,51 @@ while others (or their subtypes) can be instantiated.
+| [AbiSpecificInteger][]                                                                                        | The supertype of all ABI-specific integer types.             |
+| [Char]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Char-class.html)                         | Represents the C `char` type.                                |
+| [Int]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Int-class.html)                           | Represents the C `int` type.                                 |
+| [IntPtr]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/IntPtr-class.html)                     | Represents the C `intptr_t` type.                            |
+| [Long]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Long-class.html)                         | Represents the C `long int` (`long`) type.                   |
+| [LongLong]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/LongLong-class.html)                 | Represents the C `long long` type.                           |
+| [Short]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Short-class.html)                       | Represents the C `short` type.                               |
+| [SignedChar]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/SignedChar-class.html)             | Represents the C `signed char` type.                         |
+| [Size]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/Size-class.html)                         | Represents the C `size_t` type.                              |
+| [UintPtr]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/UintPtr-class.html)                   | Represents the C `uintptr_t` type.                           |
+| [UnsignedChar]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/UnsignedChar-class.html)         | Represents the C `unsigned char` type.                       |
+| [UnsignedInt]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/UnsignedInt-class.html)           | Represents the C `unsigned int` type.                        |
+| [UnsignedLong]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/UnsignedLong-class.html)         | Represents the C `unsigned long int` (`unsigned long`) type. |
+| [UnsignedLongLong]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/UnsignedLongLong-class.html) | Represents the C `unsigned long long` type.                  |
+| [UnsignedShort]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/UnsignedShort-class.html)       | Represents the C `unsigned short` type.                      |
+| [WChar]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-ffi/WChar-class.html)                       | Represents the C `wchar_t` type.                             |
 {:.table .table-striped }
 
 #### Instantiable native types
```

Comment:
I'd have this above the ABI-specific types, they are more important. Developers want to learn about `Pointer` earlier rather than later.